src/librustdoc/html/render/mod.rs

```diff
@@ -812,17 +812,17 @@ fn assoc_type(
 
 fn render_stability_since_raw(
     w: &mut Buffer,
-    ver: Option<&str>,
+    ver: Option<Symbol>,
     const_stability: Option<&ConstStability>,
```

Comment:
Can you remove the `&` here? `ConstStability` is Copy AFAICT. (But in a separate commit please.)

Reply:
Done as well.